### PR TITLE
Drop glade/not-implemented.glade

### PR DESCRIPTION
**Pull request by oskarb**
It relies on deprecated GnomeMessageBox, and frankly it's so small that it should just be a helper method instead of a glade-file.

Anyway, it was only used for a small disabled piece of code in journal.c, that we might as well remove.

### CODE REVIEW

**Comment by oskarb** [comment]
This removes the last reference to GnomeMessageBox, resolving one part of #13.

File: src/journal.c [hunk 1]
```diff
@@ -604,15 +604,6 @@ static void task_popup_cb(Wiggy *wig)
...
 
-/* ============================================================== */
-
-#if LATER
-static void on_print_clicked_cb(GtkWidget *w, gpointer data)
-{
-    GladeXML *glxml;
-    glxml = gtt_glade_xml_new("glade/not-implemented.glade", "Not Implemented");
-}
-#endif
```
Comment (markuspg) [inline src/journal.c:607]: Please drop this empty line, otherwise there are two successive empty lines (`clang-format` complaint).

**Comment by markuspg** [comment]
You may call me pedantic, but my aim is to port GnoTime as verbatim as possible (before conducting definitely recommended refactoring - and extensions - later). For that reason I would prefer to keep `on_print_clicked_cb` and its signal connection around, even if they are disabled right now. Contrary to the calendar functionality that was totally dysfunctional I would hope that the addition of printing should be relatively cheap when GnoTime got upgraded to a 2020s environment.

So feel free to merge as-is, but if I may say my preference would be to drop just the lines `612` and `613` of `journal.c` and keeping the functions themselves as reminders around.

**Review by markuspg — APPROVED** [review]

**Comment by oskarb** [comment]
@markuspg To avoid another review cycle for a minor change, I went ahead and merged as is. I intend to account for your suggestions when I rebase the journal.c glade modernization branch.

**Comment by markuspg** [comment]
Perfectly fine :-)

**Pull request merged**